README.md

```diff
@@ -2812,6 +2812,26 @@ class Overwriter < Grape::Middleware::Base
 end
 ```
 
+You can add your custom middleware with `use`, that push the middleware onto the stack, and you can also control where the middleware is inserted using `insert`, `insert_before` and `insert_after`.
+
+```ruby
+class CustomOverwriter < Grape::Middleware::Base
+  def after
+    [200, { 'Content-Type' => 'text/plain' }, [@options[:message]]]
+  end
+end
+
+
+class API < Grape::API
+  use Overwriter
+  insert_before Overwriter, CustomOverwriter, message: 'Overwrited again.'
+  insert 0, CustomOverwriter, message: 'Overwrited definitive.'
```

Comment:
Overwritten definitive doesn't sound great in English, how about "Overwrites all other middleware."?